frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -66,6 +66,19 @@ class Multiselect extends React.PureComponent<Props, State> {
     value: this.initialValue,
   }
 
+  public overMaxSelections(): boolean {
+    return (
+      this.props.element.maxSelections > 0 &&
+      this.state.value.length >= this.props.element.maxSelections
+    )
+  }
+
+  public getNoResultsMsg(): string {
+    const option =
+      this.props.element.maxSelections !== 1 ? "options" : "option"
+    return `You can only select up to ${this.props.element.maxSelections} ${option}. Remove an option first.`
+  }
```

Comment:
nit: We're not terribly careful about how we define method visibility throughout our frontend components, but these can be made private since I don't think we'll ever want to be able to call them externally